src/components/form-datepicker/form-datepicker.js

```diff
@@ -390,9 +390,7 @@ export const BFormDatepicker = /*#__PURE__*/ Vue.extend({
       return h(scope.isHovered || scope.hasFocus ? BIconCalendarFill : BIconCalendar, data)
     }
     let $button = h('div', { attrs: { 'aria-hidden': 'true' } }, [
-      this.hasNormalizedSlot('button-content')
-        ? this.normalizeSlot('button-content', btnScope)
-        : defaultButtonFn(btnScope)
+      this.normalizeSlot('button-content', btnScope) || defaultButtonFn(btnScope)
```

Comment:
The only issue with using `||` instead of a ternary, is that if the slot returns `''` it will render the default icons.

Reply:
@tmorehouse Hmm, haven't thought about that.

Reply:
We can always leave as is for now, and people have issues we can use the `hasNormalizedSlot` check.